_posts/2017-10-10-PSR4.md

```diff
@@ -41,14 +41,14 @@ class Foo {
 
 ```
 
-Soubor se musí jmenovat stejně jako třída a patří do složky `ApplicationOne`, protože je v namespacu `ApplicationOne`. Může obsahovat jen tuto třídu a nic jiného. 
+Soubor se musí jmenovat stejně jako třída a patří do složky `ApplicationOne`, protože je v namespace `ApplicationOne`. Může obsahovat jen tuto třídu a nic jiného. 
 
-Namespaces lze i vnořovat; bude li tedy třída mít namespace `ApplicationOne\ModuleUser` a bude se jmenovat `UserDatabaseHandler`, soubor s ní by se měl jmenovat `UserDatabaseHandler.php` a být ve složce `ModuleUser`, která bude v další složce `ApplicationOne`, kde bude i celý zbytek aplikace. 
+Namespaces lze i vnořovat; bude li tedy třída mít namespace `ApplicationOne\ModuleUser` a bude se jmenovat `UserDatabaseHandler`, soubor s ní by se měl jmenovat `UserDatabaseHandler.php` a být zanořen ve složce `ApplicationOne/ModuleUser`. 
 
 Pozor - jako většina PHP jsou i namespaces **citlivé** na **velikost písmen** (case-sensitive)!
 
 ## Zprovoznění
-Pro použití standardu PSR-4 je více než vhodné mít Composer [(návod na blogu tady)](http://jakpsatphp.cz/Jak-psat-php-Zavislosti+verzovaci-systemy+debugging/#composer). 
```

Comment:
Díky za opravení češtiny i angličtiny! Jedinnou poznámku mám tu - co tento řádek upravit na kratší `...více než vhodné mít [Composer](http://jakpsatphp.cz/Jak-psat-php...`

Reply:
Taková textace vytváří dojem, že odkaz vede na stránku Composeru. Ale pokud je to článek na tomto webu, přijde mi vhodnější toto. 

Mohl by se ještě uvést název toho odkazovaného článku, ale ten je v tomto případě ještě více zavádějící pro čtenáře, který neví, kam odkaz vede.

Reply:
Je to tak - proto jsem zvolil původní formulaci. Vycházím z předpokladu, že když někdo čtenář chtít zjistit, co je Composer, rovnou mu naservíruju blogpost s článkem, než homepage composeru. Navíc si trochu provážu blogposty navzájem (:

Reply:
Však jo, to použití je v daném kontextu ujde. Jen jsem ho přeformuloval, aby to bylo psáno profesionálnější češtinou (bo užívat v odkazech jako nosné zájmeno slovo „tady“ nebo „zde“ je lame.

Reply:
OK, už se asi chápeme. Ještě jednou díky!